src/main/java/seedu/address/ui/CalendarView.java

```diff
@@ -0,0 +1,227 @@
+package seedu.address.ui;
+
+import java.time.DayOfWeek;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Comparator;
+import java.util.logging.Logger;
+
+import javafx.collections.FXCollections;
+import javafx.collections.ListChangeListener;
+import javafx.collections.ObservableList;
+import javafx.fxml.FXML;
+import javafx.scene.control.Label;
+import javafx.scene.control.ListCell;
+import javafx.scene.control.ListView;
+import javafx.scene.layout.Region;
+import seedu.address.commons.core.LogsCenter;
+import seedu.address.logic.Logic;
+import seedu.address.model.session.RecurringSession;
+import seedu.address.model.session.Session;
+import seedu.address.model.session.SessionDate;
+import seedu.address.model.student.Student;
+import seedu.address.model.tuition.Tuition;
+
+public class CalendarView extends UiPart<Region> {
+    private static final Logger logger = LogsCenter.getLogger(CalendarView.class);
+    private static final String FXML = "CalendarView.fxml";
+
+    private Logic logic;
+    private ObservableList<Tuition> tuitionList;
+
+    // must be a monday, change through setStartDate()
```

Comment:
Maybe can use "Must" instead of "must" for consistency.

Reply:
got it!